DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -140,6 +140,26 @@ public function process(ContainerBuilder $container)
                     $groups[$resolvedGroupName]['roles'] = $groupDefaults[$resolvedGroupName]['roles'];
                 }
             }
+        } elseif ($container->getParameter('sonata.admin.configuration.admins_sort')) {
+            $groups = $groupDefaults;
+
+            // sort the default group definition
+            ksort($groups);
+            array_walk(
```

Comment:
to sort all the elements of each group by their label - If you have a better idea how to achieve that, please :)